std/stdio.d

```diff
@@ -2236,22 +2344,23 @@ $(XREF file,readText)
 
     unittest
     {
-        // static import std.file;
-        //
-        // auto deleteme = testFilename();
-        // rndGen.popFront();
-        // scope(failure) printf("Failed test at line %d\n", __LINE__);
-        // std.file.write(deleteme, "1 2\n4 1\n5 100");
-        // scope(exit) std.file.remove(deleteme);
-        // File f = File(deleteme);
-        // scope(exit) f.close();
-        // auto t = [ tuple(1, 2), tuple(4, 1), tuple(5, 100) ];
-        // uint i;
-        // foreach (e; f.byRecord!(int, int)("%s %s"))
-        // {
-        //     //.writeln(e);
-        //     assert(e == t[i++]);
-        // }
+         import std.file;
+
+         auto deleteme = testFilename();
```

Comment:
Use of `testFilename` again. Also probably shouldn't use `printf` in the line following, use writefln. Use e.g. `name` instead of deleteme.